heidegger_index/management/commands/populate_index.py

```diff
@@ -90,7 +90,11 @@ def handle(self, *args, **kwargs):
         lemma_objs = dict()
         sort_keys = dict()
         for i, (value, data) in enumerate(index_data.items()):
-            lemma_obj = Lemma(id=i, value=value, type=data.get("type", None))
+            try:
+                md = data["metadata"]
+            except:
```

Comment:
Maakt nu misschien niet heel veel uit, maar het is een goede gewoonte om expliciet te zijn over welke error je verwacht. Je kunt onverwachte errors anders over het hoofd zien.

Reply:
Overigens kun je ook `md = data.get("metadata", {})` gebruiken.